sdk/servicebus/Microsoft.Azure.WebJobs.Extensions.ServiceBus/src/Primitives/ServiceBusReceivedMessageExtensions.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Messaging.ServiceBus;
+
+namespace Microsoft.Azure.WebJobs.ServiceBus
+{
+    // can be removed once the SDK is upgraded to the version that supports the message state
```

Comment:
Your PR https://github.com/Azure/azure-sdk-for-net/pull/25446 is merged. So I think we should just pull in the updated version?

Reply:
If it is released in the meantime I can do that.

Reply:
7.6 is about to be released but not yet entirely. As soon as it is out I can bump the package reference and delete the code

Reply:
7.6 was released yesterday. I updated the PR